modules/ibm-z-secure-execution.adoc

```diff
@@ -0,0 +1,104 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z-kvm.adoc
+// * installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc
+
+:_content-type: PROCEDURE
+[id="installing-rhcos-using-ibm-secure-execution_{context}"]
+= Installing {op-system} using IBM Secure Execution
+
+Before you install {op-system} using IBM Secure Execution, you must prepare the underlying infrastructure.
+
+:FeatureName: Installing {op-system} using IBM Secure Execution
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* IBM z15 or later, or IBM LinuxONE III or later
+* {op-system-base-full} 8 or later
```

Comment:
```suggestion
* {op-system-base-full} 8 or later.
```

Reply:
I think because this list contains fragments and complete sentences you have to have `.` after everyone one. https://www.ibm.com/docs/en/ibm-style?topic=punctuation-periods#lists

Reply:
I have to admit I'm never 100% sure about those you see a mix allover the place 🙈